plexpy/activity_handler.py

```diff
@@ -83,8 +85,10 @@ def on_stop(self, force_stop=False):
             db_session = ap.get_session_by_key(session_key=self.get_session_key())
 
             # Fire off notifications
-            threading.Thread(target=notification_handler.notify,
-                             kwargs=dict(stream_data=db_session, notify_action='stop')).start()
+            progress_percent = helpers.get_percent(self.timeline['viewOffset'], db_session['duration'])
+            if progress_percent < plexpy.CONFIG.NOTIFY_WATCHED_PERCENT:
```

Comment:
What if we're not using watched notifications and rely on stopped notifications or if we want both?

These notification changes should be something we can toggle - one setting toggle is fine with me. Something like `permit consecutive notifications` which you can disable to achieve this.